opflexagent/as_metadata_manager.py

```diff
@@ -391,7 +410,8 @@ def process(self, files):
 class StateWatcher(FileWatcher):
     def __init__(self):
         root_helper = cfg.CONF.AGENT.root_helper
-        self.mgr = AsMetadataManager(LOG, root_helper)
+        bridge_manager = get_bridge_manager(cfg.CONF.OPFLEX)
```

Comment:
I don't think this will work - this is just the class, and the AsMetadataManager is expecting an instance. Also, the plug method requires that the initialize method on the bridge_manager has been called. I think we need a different approach for all of this.

Reply:
Have fixed this.